### ISSUE TITLE
Rename Chrome to Chromium in main page "Browser scores"

### ISSUE THREAD

**Issue by blama** [Fyrd/caniuse#6027]
Now that IE and IE Edge is not shown here any more, it should say "Chromium" instead of "Chrome" in order to also include IE.

![grafik](https://user-images.githubusercontent.com/6358308/133764596-b5f18abf-c6b2-4a55-9f96-d781a47d8b41.png)

**Comment by atjn** [comment]
Hmmm, writing "Chromium" would also be misleading, as there are differences between Chrome and Edge, and the bar would have to track one or the other. Chrome is still by far the most used browser, but i guess Fyrd could consider adding Edge here as well 🤷

**Comment by blama** [comment]
> Hmmm, writing "Chromium" would also be misleading, as there are differences between Chrome and Edge, and the bar would have to track one or the other. Chrome is still by far the most used browser, but i guess Fyrd could consider adding Edge here as well 🤷

Oh, what are the differences (apart from slightly delayed releases)?
But I agree, if there are differences, both should be listed.

**Comment by atjn** [comment]
We do occasionally see differences between Edge and Chrome. The only issue I can think of right now, is support for AVIF: https://caniuse.com/avif

**Comment by Fyrd** [comment]
Yeah, it's a little difficult to decide here since there are a few minor differences between Chrome, Edge and even between Chrome and Chromium (which is why I'd rather not change it to Chromium as that would be inaccurate).

The list is basically all tracked desktop browsers that use their own engine, that's why Chrome gets to be there but not Edge anymore. Also if I included Edge I should technically include Opera and others as well and the data would be near-identical anyway so that's why I settled on this list.
